### PR TITLE
Added docker environment for easier setup

### LINKED ISSUE #174: Dockerize this thing
Because.

### PR DESCRIPTION
**Pull request by asgrim**
Fixes #174

### CODE REVIEW

**Comment by asgrim** [comment]
@jezemery poke

**Comment by lboynton** [comment]
We might as well update the local.php config file with the docker DB credentials, mightn't we?

File: docker/postgres/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM postgres:9.6.3
```
Comment (lboynton) [inline docker/postgres/Dockerfile]: This file seems a bit redundant, could simply specify the image in the docker-compose.yml file

File: run-in-php-docker.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+args="$@"
+docker exec -ti phphsite_phph-php-fpm_1 bash -c "cd /app; $args"
```
Comment (lboynton) [inline run-in-php-docker.sh]: This doesn't work unless your user is in the docker group. Might be better just to use `docker-compose exec phph-php-fpm <command>`?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=phphants
+    ports:
+      - 5432:5432
+
+  phph-php-fpm:
+    build: docker/php-fpm
+    volumes:
+      - .:/app
+    depends_on:
```
Comment (lboynton) [inline docker-compose.yml]: As we have a `links` entry, this is redundant

File: docker/postgres/Dockerfile (hunk 1 above)
Comment (lboynton) [reply docker/postgres/Dockerfile]: I'm using this file now :)

Comment (jezemery) [reply docker/postgres/Dockerfile]: <3 @lboynton that fixed the problems I was having!

File: README.md [hunk 4]
```diff
@@ -10,21 +10,46 @@ For more information, visit us at [PHP Hampshire](http://phphants.co.uk).
...
 ```bash
-git clone https://github.com/phphants/phph-site.git
-cd phph-site
-composer install
 cp config/autoload/local.php.dist config/autoload/local.php
-# Edit the local.php config to suit your environment
-vendor/bin/doctrine-migrations migrate
 ```
 
+Edit the configuration file accordingly with API tokens.
```
Comment (jezemery) [inline README.md]: Probably worth adding here that the postgres connection string needs putting into the config file too?

**Pull request merged**